packages/z/zotero/files/zotero.desktop

```diff
@@ -1,9 +1,10 @@
 [Desktop Entry]
 Name=Zotero
 Comment=Collect, organize and cite research
-Exec=zotero
+Exec=/usr/bin/zotero
```

Comment:
Why are you using a full path `Exec=/usr/bin/zotero` instead of `Exec=zotero`? Did you had any issue with it?